packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -822,13 +824,7 @@ export class BaseResolversVisitor<
         // Make sure the inner type has no ResolverTypeWrapper
         const name = clearWrapper(isScalar ? this._getScalar(typeName) : prev[typeName]);
         const replaced = replacePlaceholder(this.config.defaultMapper.type, name);
-
-        // Don't wrap Union with ResolverTypeWrapper, each inner type already has it
-        if (isUnionType(schemaType)) {
-          prev[typeName] = replaced;
-        } else {
-          prev[typeName] = applyWrapper(replacePlaceholder(this.config.defaultMapper.type, name));
-        }
```

Comment:
We now handle all union logic in `ResolversUnionTypes` so there's no need for special treatments here

Reply:
This comment is no longer relevant due to change in implementation details